doc/classes/Object.xml

```diff
@@ -10,11 +10,18 @@
 		Some classes that extend Object add memory management. This is the case of [Reference], which counts references and deletes itself automatically when no longer referenced. [Node], another fundamental type, deletes all its children when freed from memory.
 		Objects export properties, which are mainly useful for storage and editing, but not really so much in programming. Properties are exported in [method _get_property_list] and handled in [method _get] and [method _set]. However, scripting languages and C++ have simpler means to export them.
 		Property membership can be tested directly in GDScript using [code]in[/code]:
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var n = Node2D.new()
 		print("position" in n) # Prints "True".
 		print("other_property" in n) # Prints "False".
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		var node = new Node2D();
+		GD.Print(node.Get("position") != null);
+		GD.Print(node.Get("other_property") != null);
+		[/csharp]
```

Comment:
The doc you're editing is specifically about the `in` operator... I don't see how this C# codeblock documents `in`.

Reply:
We don't have the ability in C# so I wrote a workaround. Maybe I should add something like
`// C# has no built-in in support, but we can achieve same behaviour by performing a null check.`

Reply:
Sounds good yeah.
```
// C# has no equivalent `in` operator support, but we can achieve that behavior by performing a null check.
```

Reply:
Here's my suggestion:

```suggestion
		[csharp]
		var node = new Node2D();
		// C# has no direct equivalent to GDScript's `in` operator here, but we
		// can achieve the same behavior by performing `Get` with a null check.
		GD.Print(node.Get("position") != null); // Prints "True".
		GD.Print(node.Get("other_property") != null); // Prints "False".
		[/csharp]
```